lib/identity_cache/replicated_cache_proxy.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module IdentityCache
+  class ReplicatedCacheProxy
+    def initialize(cache_backend, blob_replication_factor:)
+      @cache_backend = cache_backend
+      @blob_replication_factor = blob_replication_factor
+    end
+
+    def write(key, value)
```

Comment:
I don't think `IdentityCache.cache.write` is ever used, so maybe we should just remove that and avoid having to support at each proxy layer (i.e. we could remove the write method from MemoizedCacheProxy, CacheFetcher and FallbackFetcher).